File: cprograms/Npowerseries.c
#include <stdio.h>

int main(void) {
  int LastTerm, Counter, Result = 1, Number = 1, Count;
  printf("Enter the number upto which you want the series:");
  scanf("%d", &LastTerm);
  printf("The series upto %d is", LastTerm);
  for(; Result <= LastTerm; )
  {
    Result = 1;
    for(Counter = 1; Counter <= Number; Counter++)
    {
       Result = Result * Number;
    }
    if(Result <= LastTerm)
    {
    printf(" %d", Result);
    Number++;
    }
  }
  printf(".");
  return 0;
}

File: javascriptprograms/Npowerseries.js
var readln = (typeof readline === 'function') ? (readline) : (function() {
     importPackage(java.io);
     importPackage(java.lang);
     var stdin = new BufferedReader(new InputStreamReader(System['in']));

     return function() {
         return String(stdin.readLine());  // Read line, 
     };                                    // force to JavaScript String
 }());
System.out.print("How many terms you want in the series?");
NoOfTerms = parseInt(readln());
var Number = 1;
var Counter = 1;
var Result = 1;
var Series = " ";
for (Counter = 1; Counter <= NoOfTerms; Counter++){
  Counter = 1;
  while(Counter <= Number){
    Result = Result * Number;
    Counter = Counter + 1;
  }
  Counter = Counter - 1;
  if(Counter <= NoOfTerms){
    Series += " " +Result;
    Number = Number + 1;
    Result = 1;
  }
}
print("The " + NoOfTerms + " terms in the series are" + Series + ".");
      